.github/workflows/test-tutorial.yml

```
@@ -0,0 +1,10 @@
+name: Test GM Tutorial
+on: workflow_dispatch
```

we can start with the `push` event, as that will trigger on all commits. 
```suggestion
on: push
```